### PR TITLE
Pipelines as Code documentation for Pipelines 1.9 | Peer-review comment updates

### PR DESCRIPTION
**Pull request by gtrivedi88**
**Original PR:** https://github.com/openshift/openshift-docs/pull/53813
OCP version for cherry-picking: `enterprise-4.10`, `enterprise-4.11`, `enterprise-4.12`, `enterprise-4.13`
**SME Review**: Completed
**QE Review:** Completed

**Peer-Review:** Comments addressed in this PR

### CODE REVIEW

File: modules/op-creating-pipeline-run-using-pipelines-as-code.adoc [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    event == "pull_request" && target_branch != experimental"
+...
+----
+
+For advanced CEL-based filtering while using {pac}, you can use the following fields and suffix functions:
+
+* `event`: A `push` or `pull_request` event.
+* `target_branch`: The target branch.
+* `source_branch`: The branch of origin of a `pull_request` event. For `push` events, it is same as the `target_branch`.
+* `event_title`: Matches the title of the event, such as the commit title for a `push` event, and the title of a pull or merge request for a `pull_request` event. Currently, only GitHub, Gitlab, and BitbucketCloud are the supported providers.
```
Comment (shipsing) [inline modules/op-creating-pipeline-run-using-pipelines-as-code.adoc:133]:
```suggestion
* `event_title`: Matches the title of the event, such as the commit title for a `push` event, and the title of a pull or merge request for a `pull_request` event. Currently, only GitHub, Gitlab, and Bitbucket Cloud are the supported providers.
```

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@gtrivedi88 There was a huge number of review comments. Thanks for your effort in addressing all the changes and suggestions requested. Some of them were missed out - PTAL at my suggestions. Thanks again!

**Review by Srivaralakshmi — APPROVED** [review]
@gtrivedi88: Thanks for the changes. Approved!

**Comment by shipsing** [comment]
> tkn-pac

@gtrivedi88 There is still an instance of `tkn-pac` in this section: https://57455--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/using-pipelines-as-code.html#pac-key-features.

Please check. Rest all looks good.

**Review by shipsing — APPROVED** [review]
/lgtm

**Pull request merged**